app/controllers/album.js

```diff
@@ -0,0 +1,32 @@
+const albumService = require('../services/albumService');
+const log = require('../logger');
+
+module.exports = {
+  getAlbums: (req, res, next) => {
+    albumService
+      .getAlbums()
+      .then(response => response.json())
```

Comment:
you can execute .json function in service directly to encapsulate behavior of albumsService